lib/core/src/Cardano/Wallet/Primitive/Fee.hs

```diff
@@ -82,6 +83,10 @@ data TxSizeLinear =
 cardanoPolicy :: TxSizeLinear
 cardanoPolicy = TxSizeLinear (Quantity 155381) (Quantity 43.946)
 
+-- | Hard-coded zero fee policy for Jormungandr
+jormungandrPolicy :: TxSizeLinear
```

Comment:
👍 

I am a _tiny_ bit worried that our test setup will differ from `exe` setup _eventually_, but that's not just about the fee policy

Reply:
Yeah, ideally, this should come from the genesis block for Jörmungandr. I'll make a PR A.S.A.P